### DISCUSSION TITLE
'Create  list' window stuck

### DISCUSSION THREAD

**Discussion by sstiefel19** [community/community#129154] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

wanted to create a list but after clicking  the create button, the window is stuck for 10 minutes now.
<img width="536" alt="Screenshot 2024-06-20 at 08 50 53" src="https://github.com/community/community/assets/55794986/72992f07-9db9-42a7-8a01-5e87754ae840">

**Comment by sovetski** [comment]
Same here, if I reload the page I see the new list

**Comment by azizmobarak** [comment]
i confirm , same here

**Reply by Gogo45345popo** [reply]
@popo same time

**Comment by 1alexbc1** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> wanted to create a list but after clicking  the create button, the window is stuck for 10 minutes now.
> <img width="536" alt="Screenshot 2024-06-20 at 08 50 53" src="https://github.com/community/community/assets/55794986/72992f07-9db9-42a7-8a01-5e87754ae840">
>

**Comment by 1alexbc1** [comment]
> i confirm , same here

**Comment by hthief** [comment]
Same for me, the modal can be closed by clicking on the background tho

**Comment by paullferguson** [comment]
Love the idea.
After hard refreshed in Chrome 127 OSX 14.5
For me, it opens with inputs and the CTA button in a disabled state.
The feedback link works but it didn't close the modal when it navigated to this forum.
Also not dismissed by the `esc` key.

**Comment by vykrant** [comment]
Browsing around pages but the iframe is stuck

**Comment by Gits-hub** [comment]
yeah, had the same problem.

**Comment by jhj0517** [comment]
This happens when I try to create a list in the repository that I haven't starred.

To reproduce the error, 
1. Go to the repository that you didn't star

2. Click the "Create list" button, then the focus should be on the "Name the list" text field, but the focus is on the repository searchbar behind.

![image](https://github.com/user-attachments/assets/743d3ae1-5568-437f-be5f-b0b34db0e91b)


When I tried in the starred repository, the "Create list" button works fine.

- Broswer
    -   Microsoft Edge
         version 127.0.2651.86

**Comment by kamallahloh** [comment]
Same with me and even when I moved to the report page the modal won't close automatically.
![Screenshot 2024-08-19 145057](https://github.com/user-attachments/assets/914636a4-574c-43a9-a62b-266457c94a58)
![Screenshot 2024-08-19 145124](https://github.com/user-attachments/assets/e73d2f48-e41e-413d-b593-24ea008b4cfd)
